### PR TITLE
Islam/dx 370 add manual client setup to axiom py in

### PR DESCRIPTION
**Pull request by schehata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,13 +45,32 @@ Create and use a client like this:
...
+You can also configure the client using options passed to the client constructor:
+
+```py
+import os
+import axiom
+
+access_token = os.getenv("AXIOM_TOKEN")
+org_id = os.getenv("AXIOM_ORG_ID")
+
+client = axiom.Client(access_token, org_id)
```
Comment (bahlo) [inline README.md:71]:
```suggestion
import axiom

client = axiom.Client("<api token>", "<org id>")
```

And can we move this above the example code block like in axiom-go?

File: axiom/client.py [hunk 2]
```diff
@@ -71,11 +167,162 @@ def __init__(self, token: str, org_id: str = None, url_base: str = AXIOM_URL):
...
+        params = self._prepare_query_options(opts)
+        res = self.session.post(path, data=payload, params=params)
+        result = Util.from_dict(QueryLegacyResult, res.json())
+        self.logger.debug(f"query result: {result}")
+        query_id = res.headers.get("X-Axiom-History-Query-Id")
+        self.logger.info(f"received query result with query_id: {query_id}")
+        result.savedQueryID = query_id
+        return result
+
+    def apl_query(self, apl: str, opts: AplOptions) -> QueryResult:
```
Comment (bahlo) [inline axiom/client.py:244]:
```suggestion
    def query(self, apl: str, opts: AplOptions = {}) -> QueryResult:
```

File: axiom/client.py [hunk 3]
```diff
@@ -71,11 +167,162 @@ def __init__(self, token: str, org_id: str = None, url_base: str = AXIOM_URL):
...
+        # encode request payload to NDJSON
+        content = ndjson.dumps(events).encode("UTF-8")
+        gzipped = gzip.compress(content)
+
+        return self.ingest(
+            dataset, gzipped, ContentType.NDJSON, ContentEncoding.GZIP, opts
+        )
+
+    def query_legacy(
+        self, id: str, query: QueryLegacy, opts: QueryOptions
```
Comment (bahlo) [inline axiom/client.py:223]:
```suggestion
        self, id: str, query: QueryLegacy, opts: QueryOptions = {}
```

**Review by bahlo — APPROVED** [review]
nice work, some small suggestions

File: axiom/client.py (hunk 2 above)
Comment (schehata) [reply axiom/client.py:244]: there is already a function called query, we kept `apl_query` to maintain backwards compatibility, the `apl_query()` points to `query()`

**Comment by schehata** [comment]
@bahlo fixed and resolved all comments, can you take another look?

**Review by bahlo — APPROVED** [review]

**Pull request merged**